### PR TITLE
popup/view/MnemonicPhrase: Reskin ConfirmMnemonicPhrase

### PR DESCRIPTION
**Pull request by acharb**
**WHAT**
Reskin ConfirmMnemonicPhrase

Screen Shot 2022-02-02 at 4 02 27 PM

### CODE REVIEW

File: extension/src/popup/components/mnemonicPhrase/CheckButton.tsx [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 import styled from "styled-components";
 import { Field } from "formik";
-import { COLOR_PALETTE } from "popup/constants/styles";
 
 type CheckButtonProps = {
   onChange: (e: any) => void;
   wordKey: string;
   word: string;
 };
 const ButtonLabelEl = styled.label`
```
Comment (quietbits) [inline extension/src/popup/components/mnemonicPhrase/CheckButton.tsx:10]: Should we move these styled-component styles to a CSS file?

File: extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    selectedWords.map((word) => convertToWord(word)).join(" ");
+
+  if (applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) {
+    return <Redirect to={ROUTES.mnemonicPhraseConfirmed} />;
+  }
+
+  return (
+    <>
+      <FullscreenStyle />
+      <div className="ConfirmMnemonicPhrase--screen">
```
Comment (quietbits) [inline extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx:70]: NIT, in [BEM naming convention ](http://getbem.com/naming/), `--` is for a modifier. I think in most of these cases we are using them for block elements `__`. It's not like we're 100% consistent everywhere. :sweat_smile:

Comment (acharb) [reply extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx]: til! https://github.com/stellar/freighter/pull/359/commits/4a93c073f0e2d410ff35585a1da3f851e41484c4

File: extension/src/popup/components/mnemonicPhrase/CheckButton.tsx (hunk 1 above)
Comment (acharb) [reply extension/src/popup/components/mnemonicPhrase/CheckButton.tsx]: yes good point https://github.com/stellar/freighter/pull/359/commits/1e9f8c95ac98a3568b3f2e8caadbd8a9b1002856

**Review by quietbits — APPROVED** [review]

**Pull request merged**